docs/index.html

```diff
@@ -7,8 +7,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
   <meta name="description" content="Self-hosted CI/CD solution">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
-  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/lib/themes/vue.css">
-  <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/prismjs@1.23.0/themes/prism.css"></script>
```

Comment:
Just checked and diffs looks fine

![image](https://user-images.githubusercontent.com/2477952/111746312-27d5f900-888e-11eb-9b28-e070e08787d5.png)

Resolving

Reply:
Sure, let me know if you maybe want another color for the inserted, as that's easy to change:

![image](https://user-images.githubusercontent.com/22037089/111814246-888b2300-88da-11eb-9945-2aaac17640dc.png)

Reply:
Love it! Do that!

Reply:
Ok, time to squint your eyes, which green do you prefer?

![image](https://user-images.githubusercontent.com/22037089/111815543-17e50600-88dc-11eb-99ab-0288c0b81596.png)
![image](https://user-images.githubusercontent.com/22037089/111815583-23383180-88dc-11eb-8341-75a853392cbf.png)

Reply:
#42F570 or #07ED42, because the added text should pop the most imo.

Little hard to see as I dont have the full page as context, but I think I like the latter (#07ED42) the most as it's the most saturated green, and I do love me some saturated greens.

If the rest of the page is more pastell colored as the red is here, then maybe go with the less colored one (#42F570) instead

Reply:
I like #07ED42 too, using that one.